rustport/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+use once_cell::sync::Lazy;
+use std::{ffi::c_void, sync::Mutex};
+
+use stlab::{Priority, PriorityTaskSystem};
+mod stlab;
+
+/// A static instance of the task system which is invoked through the `execute` functions below.
```

Comment:
```suggestion
/// A static instance of the task system that is invoked through the `execute` functions below.
```
s/which/that/; which is for after a comma and would make this mean the"the task system" (not the instance) is invoked.

```suggestion
/// An instance of the task system that is invoked through the `execute` functions below.
```
Don't repeat stuff like `static` that is obvious from the declaration, which must appear in the generated documentation.

```suggestion
/// A task system that is invoked through the `execute` functions below.
```
"An instance" is almost always vacuous.  

Is this not “the” task system, or is it possible and useful to make another instance?
```suggestion
/// The task system.
```

or, if this is really internal to the module, you might go with:

```suggestion
/// The shared state of the task system.
```